### ISSUE TITLE
Add documentation about localization inside `package.json`

### ISSUE THREAD

**Issue by joaomoreno** [Microsoft/vscode-nls#11]

**Comment by dbaeumer** [comment]
@joaomoreno is the Readme.md that is bundled with npm not enough? I am actually not understanding what you expect to have in the package.json.

**Comment by dbaeumer** [comment]
Got it. Added  readme property to point to README.md

**Comment by dbaeumer** [comment]
There seems to be a cache problem with README.md an npm according to SO. They suggest to add `"readme": "See README.md file"`. Would this address your concerns.

**Comment by joaomoreno** [comment]
No, I meant it as document how to localize strings in `package.json`, through `package.nls.json`.

**Comment by dbaeumer** [comment]
OK. Now I see. I will add something to the README although this is VS Code specific.
